NotesApp/Scenes/Editor/EditorViewController.swift

```diff
@@ -0,0 +1,206 @@
+//
+//  EditorViewController.swift
+//  NotesApp
+//
+//  Created by Luka Gujejiani on 01.11.24.
+//
+
+import UIKit
+
+protocol AddNoteDelegate: AnyObject {
+    func didAddNote()
+}
+
+class EditorViewController: UIViewController {
+    // MARK: - Properties
+    private let viewModel: EditorViewModel
+    weak var delegate: AddNoteDelegate?
+
+    private lazy var customBackButton = CustomBackButton(
+        iconName: "chevron.left",
+        pointSize: 20,
+        backgroundColor: .darkGray,
+        tintColor: .white,
+        cornerRadius: 14,
+        size: 48
+    )
+    
+    private lazy var titleTextView: UITextView = {
+        let textView = UITextView()
+        textView.font = UIFont.systemFont(ofSize: 24, weight: .bold)
+        textView.textColor = .white
+        textView.backgroundColor = .clear
+        textView.delegate = self
+        textView.isEditable = true
```

Comment:
remove textView.isEditable = true because the default value for isEditable is true. You can click option button on keyboard and tap on isEditable, you will see what is the default value